app/assets/javascripts/shared/datatable/main.js

```diff
@@ -305,4 +305,25 @@ class DradisDatatable {
     localStorageData.columns = newColumns;
     return localStorageData;
   }
+
+  versionedLocalStorageKey(localStorageKey) {
+    if (!localStorageKey) {
+      return;
+    }
+
+    // V2 was introduced as a result of a fix to persist user's DataTable state
+    // (see https://github.com/dradis/dradis-ce/pull/901).
+    //
+    // It adds table headers on the page to localStorage data so that
+    // when the page loads, it uses these table headers to identify if columns
+    // on the page are existing or new.
+    //
+    // But users with existing localStorage data will not have these table headers
+    // saved because it was only added in the same patch.
+    //
+    // So this is where versionedLocalStorageKey comes in. It indirectly resets
+    // the DataTable to use default columns and prevents the need for a localStorage
+    // migration.
+    return `${localStorageKey}.v2`;
```

Comment:
Can we save the version number as a variable/constant? It'll be easier to tweak it in the future that way.

Reply:
Good call!